### DISCUSSION TITLE
Any way to include the currently running time in the Reporting?

### DISCUSSION THREAD

**Discussion by gerroon** [kimai/kimai#3325] [Questions & Answers] [answered]
Hi

Is it possible to include the currently running timer in the reporting? Let's say my timer is running for 6 hours for a task, and I look at the reporting page, that time is not listed and it behaves a bit misleading if you will. I am wondering if I can me Kimai to to include the active time recording in these types of reports.

thanks

**Comment by kevinpapst** [comment, marked as answer]
No, that is not possible. Running entries will never be included in reports.
Your record is not stoped and may only run accidentally for so long, it is not reliable.
